vllm/utils.py

```diff
@@ -386,6 +374,55 @@ def create_kv_caches_with_random(
         torch_dtype = cache_dtype
     else:
         raise ValueError(f"Invalid kv cache dtype: {cache_dtype}")
+    return torch_dtype
+
+
+def create_kv_caches_with_random_flashinfer(
```

Comment:
```suggestion
def create_kv_caches_with_random_flash(
```
(since it is same for both flash infer and attn)